src/ir/analysis/derive_partial_eq_or_partial_ord.rs

```diff
@@ -115,7 +124,7 @@ impl<'ctx> MonotoneFramework for CannotDerivePartialEqOrPartialOrd<'ctx> {
         trace!("constrain: {:?}", id);
 
         if self.cannot_derive_partialeq_or_partialord.contains(&id) {
-            trace!("    already know it cannot derive PartialEq or PartialOrd");
+            trace!("    already know it cannot derive `PartialEq`/`PartialOrd`");
```

Comment:
Is it makes sense to put markdownique syntax in logs?

Reply:
I like to be consistent :)